packages/app-config-writer/src/types/smartLinks.ts

```diff
@@ -7,13 +7,12 @@ export enum DeployConfig {
     'FioriToolsServestatic' = 'fiori-tools-servestatic'
 }
 
-export enum UrlParameters {
-    UI2 = '/sap/bc/ui2/start_up?so=%2A',
-    Action = 'action=%2A',
-    Alias = 'systemAliasesFormat=object',
-    Language = 'sap-language=EN',
-    ShellType = 'shellType=FLP',
-    Depth = 'depth=0'
+export const UrlParameters = {
```

Comment:
btw, I would move this constant into the code where you use it since it is only used once and makes it easier to see directly in the code what we are adding to the request

Reply:
Ok, will move that